backend/main/models/vote.go

```diff
@@ -194,14 +201,27 @@ func (v *Vote) GetVoteById(db *s.Database) error {
 }
 
 func (v *Vote) CreateVote(db *s.Database) error {
-	err := db.Conn.QueryRow(db.Context,
-		`
-		INSERT INTO votes(proposal_id, addr, choice, composite_signatures, cid, message)
-		VALUES($1, $2, $3, $4, $5, $6)
-		RETURNING id, created_at
-	`, v.Proposal_id, v.Addr, v.Choice, v.Composite_signatures, v.Cid, v.Message).Scan(&v.ID, &v.Created_at)
+	err := createVote(db, v)
+
+	if checkError(err) {
+		return err
+	}
+
+	proposal, err := getProposal(db, v.Proposal_id)
+
+	if checkError(err) {
+		return err
+	}
+
+	err = checkForEarlyVoteAchievement(db, v, proposal)
+
```

Comment:
could you remove these blank lines before the `checkError`? So that we handle the error directly after the function call with no whitespace.

Reply:
Done.